scripts/NewTeamsAlert/src/helper/sendAlert.js

```diff
@@ -0,0 +1,63 @@
+// Function to send adaptive cards
+import dotenv from "dotenv";
+dotenv.config(); // Load environment variables from .env file
+
+import template from "../adaptiveCards/adaptive-card-template.json" assert { type: "json" };
+import { AdaptiveCards } from "@microsoft/adaptivecards-tools";
+import WebhookTarget from "../webhookTarget.js";
+import updatePendingTasks from "./updatePendingTasks.js";
+import pendingTask from "./pendingTask.js";
+import alertText from "../utils/alertText.js";
+
+// Fetch the webhook URL from environment variables
+const webhookUrl = process.env.WEBHOOK_URL;
+
+if (!webhookUrl) {
+  throw new Error("WEBHOOK_URL environment variable is not set.");
+}
+
+const webhookTarget = new WebhookTarget(new URL(webhookUrl));
+
+const sendAlert = async (tasks) => {
+  const pendingTaskCount = new Map();
```

Comment:
why do we have map name ending with Count, we only add Count in variable name where we are storing count of something, please improve variable names wherever possible